.renovaterc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "extends": [
+    ":gitSignOff",
+    "group:monorepos",
+    "group:recommended",
+    ":pinAllExceptPeerDependencies",
+    "helpers:disableTypesNodeMajor"
+  ],
+
+  "ignorePaths": [
+    "**/node_modules/**",
+    "**/__tests__/**",
+    "**/test/**"
+  ],
+
+  "automerge": false,
+  "branchPrefix": "renovate/",
+  "ignoreUnstable": true,
+  "statusCheckVerify": true,
+  "updateNotScheduled": true,
+
+  "lockFileMaintenance": {
+    "enabled": true,
+    "schedule": "before 5am on monday"
+  },
+
+  "prConcurrentLimit": 20,
+  "prCreation": "immediate",
+  "prHourlyLimit": 2,
+
+  "semanticCommits": true,
+  "semanticCommitType": "chore",
+  "semanticCommitScope": null,
```

Comment:
Can we use `{{baseDir}}` to signify PRs for `shopping` vs `recommender` packages?
```
By default you will see angular-style commit prefixes like "chore(deps):". If you wish to change it to something else like "package" then it will look like "chore(package):". You can also use parentDir or baseDir to namespace your commits for monorepos e.g. "{{parentDir}}".
```